### PR TITLE
Option to disable logging of log producer in audit log

### PR DESCRIPTION
**Pull request by marcstern**
when log level < 9.
[Issue #1069 - Marc Stern]

### CODE REVIEW

**Comment by zimmerle** [comment]
Please make sure that the JSON audit log is also covered.

**Comment by marcstern** [comment]
done

File: apache2/msc_logging.c [hunk 1]
```diff
@@ -1174,6 +1174,9 @@ void sec_audit_logger_json(modsec_rec *msr) {
             yajl_kv_bool(g, "response_body_dechunked", 1);
         }
 
+#ifdef LOG_NO_PRODUCER
+        if (msr->txcfg->debuglog_level < 9) return;
```
Comment (zimmerle) [inline apache2/msc_logging.c]: Please check this condition here. It does not seems to be doing what you aiming to do in the patch/commit description.

**Comment by zimmerle** [comment]
Hi @marcstern,

Made a comment in your patch, please verify. Before send me any updates make sure to do it in a clean tree. Please also avoid to long commit to replaced the \r with \r\n. Thanks.

File: apache2/msc_logging.c (hunk 1 above)
Comment (marcstern) [inline apache2/msc_logging.c]: Indeed, seems to be part of another patch.
Do I recommit? It seems I cannot amend (actually, I can but it detects no difference if I try)

**Comment by marcstern** [comment]
I can't see, in "Files changed", in which files \r were replaced by \r\n.
I can't find this on my local copy either.

File: apache2/msc_logging.c (hunk 1 above)
Comment (zimmerle) [inline apache2/msc_logging.c]: Yes, please. Try to amend all the commits and make this patch a single shot. addressing those comments.

**Pull request closed without merging**